go/tasks/plugins/array/awsbatch/jobs_store.go

```diff
@@ -147,8 +150,8 @@ func updateJob(ctx context.Context, source *batch.JobDetail, target *Job) (updat
 		target.Attempts = append(target.Attempts, a)
 	}
 
-	// If no job attempts are present, try to construct one from container status
-	if len(source.Attempts) == 0 {
+	// Add the "current" log stream to log links if one exists.
+	{
```

Comment:
at a glance, i don't understand why this is in a separate block.  can you add a comment?